experimental/federated_plugin/federated_plugin/ios/Classes/SwiftFederatedPlugin.swift

```diff
@@ -3,12 +3,24 @@ import UIKit
 
 public class SwiftFederatedPlugin: NSObject, FlutterPlugin {
   public static func register(with registrar: FlutterPluginRegistrar) {
-    let channel = FlutterMethodChannel(name: "federated_plugin", binaryMessenger: registrar.messenger())
+    let channel = FlutterMethodChannel(name: "battery", binaryMessenger: registrar.messenger())
     let instance = SwiftFederatedPlugin()
     registrar.addMethodCallDelegate(instance, channel: channel)
   }
 
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
-    result("iOS " + UIDevice.current.systemVersion)
+    guard  call.method == "getBatteryLevel"  else {
```

Comment:
nit: it looks like there are some extra whitespace characters here. Is that intentional?

Reply:
Opps! I'll fix this in the next commit for MacOS support.